File: loglib-java/src/main/java/com/robog/loglib/LogBean.java
package com.robog.loglib;

/**
 * Created by yuxingdong on 2018/6/21.
 */
public class LogBean {

    public int id;

    /**
     * Log等级
     */
    public int priority;

    /**
     * Log的TAG
     */
    public String tag;

    /**
     * Log信息
     */
    public String message;

    /**
     * Log时间
     */
    public String time;

    /**
     * 堆栈信息
     */
    public String stackTrace;

    LogBean next;

    @Override
    public String toString() {
        return id + " -> " + time + " " + tag + ": " + message;
    }
}

File: loglib/src/main/java/com/robog/loglib/LogBean.kt
package com.robog.loglib

/**
 * Created by yuxingdong on 2018/6/29.
 */
data class LogBean(
        var id: Int = 0,
        var priority: Int = 0,
        var tag: String = "",
        var message: String = "",
        var time: String = "",
        var stackTrace: String = "",
        var next: LogBean?) {

    override fun toString(): String {
        return "$id -> $time $tag: $message"
    }
}